src/figuresClasses.ts

```diff
@@ -1,19 +1,65 @@
 export interface Figure {
+  shape: 'triangle' | 'circle' | 'rectangle'
+  color: 'red' | 'green' | 'blue'
 
+  getArea(): number;
 }
 
-export class Triangle {
+type Color = 'red' | 'green' | 'blue';
+type Shape = 'triangle' | 'circle' | 'rectangle';
```

Comment:
instead of `type` you can create an `enum` here and use it later like `shape = Shape.rectangle`;